snippets/Google.Storage.V1.Snippets/StorageClientSnippets.cs

```diff
@@ -126,12 +126,12 @@ public void ListObjects()
         }
 
         [Fact]
-        public void DownloadFile()
+        public void DownloadObject()
```

Comment:
we are downloading not just an object, but an object that we're treating as a file, not a folder (although I went back and forth on this myself).